flaml/onlineml/autovw.py

```diff
@@ -85,100 +96,121 @@ def __init__(self,
+        logger.info("scheduler_args %s", self._scheduler_args)
+        logger.info("automl_runner_args %s", self._automl_runner_args)
         searcher = ChampionFrontierSearcher(**searcher_args)
         scheduler = ChaChaScheduler(**self._scheduler_args)
-        self._trial_runner = OnlineTrialRunner(max_live_model_num=self._max_live_model_num,
-                                               searcher=searcher,
-                                               scheduler=scheduler,
-                                               **self._automl_runner_args)
+        self._trial_runner = OnlineTrialRunner(
+            max_live_model_num=self._max_live_model_num,
+            searcher=searcher,
+            scheduler=scheduler,
+            **self._automl_runner_args
+        )
 
     def predict(self, data_sample):
-        """Predict on the input example (e.g., vw example)
+        """Predict on the input example (e.g., vw example).
 
         Args:
-            data_sample (vw_example)
+            data_sample: vw_example.
         """
         if self._trial_runner is None:
             self._setup_trial_runner(data_sample)
         self._best_trial = self._select_best_trial()
         self._y_predict = self._best_trial.predict(data_sample)
         # code for debugging purpose
-        if self._prediction_trial_id is None or \
-                self._prediction_trial_id != self._best_trial.trial_id:
+        if (
+            self._prediction_trial_id is None
+            or self._prediction_trial_id != self._best_trial.trial_id
+        ):
             self._prediction_trial_id = self._best_trial.trial_id
-            logger.info('prediction trial id changed to %s at iter %s, resource used: %s',
-                        self._prediction_trial_id, self._iter,
-                        self._best_trial.result.resource_used)
+            logger.info(
+                "prediction trial id changed to %s at iter %s, resource used: %s",
+                self._prediction_trial_id,
+                self._iter,
+                self._best_trial.result.resource_used,
+            )
         return self._y_predict
 
     def learn(self, data_sample):
-        """Perform one online learning step with the given data sample
+        """Perform one online learning step with the given data sample.
 
         Args:
-            data_sample (vw_example): one data sample on which the model gets updated
+            data_sample (vw_example): one data sample on which the model gets updated.
         """
         self._iter += 1
         self._trial_runner.step(data_sample, (self._y_predict, self._best_trial))
 
     def _select_best_trial(self):
-        """Select a best trial from the running trials accoring to the _model_select_policy
-        """
-        best_score = float('+inf') if self._model_selection_mode == 'min' else float('-inf')
+        """Select a best trial from the running trials accoring to the _model_select_policy"""
```

Comment:
`"""Select a best trial from the running trials accoring to the _model_select_policy."""`